src/test/scala/nl.knaw.dans.easy.stage/lib/EmdSpec.scala

```diff
@@ -58,7 +59,9 @@ class EmdSpec extends FlatSpec with Matchers with Inside with CanConnectFixture
     for (bag <- new File("src/test/resources/dataset-bags").listFiles()) {
       sdoSetDir.mkdirs()
       implicit val s: Settings = newSettings(bag)
-      EMD.create(sdoSetDir) shouldBe a[Success[_]]
+      (bag, EMD.create(sdoSetDir)) should matchPattern { // TODO use behave like
```

Comment:
needed quick trouble shooting to figure out which bag failed. Refactoring to `behave like` is to far beyond the scope of the issue.